pkg/pillar/zedcloud/send.go

```diff
@@ -348,12 +378,20 @@ func SendOnIntf(ctx ZedCloudContext, destUrl string, intf string, reqlen int64,
 				} else {
 					errorList = append(errorList, err)
 				}
+			} else if isCertUnknownAuthority(err) {
+				if usedProxy {
+					log.Errorf("client.Do fail: CertUnknownAuthority with proxy")
+				} else {
+					log.Errorf("client.Do fail: CertUnknownAuthority") // could be transparent proxy
+				}
```

Comment:
Would it make sense to define two separate Enum returns for these two cases? That way we can count them separately as well. [For next sprint.]

Reply:
ok.